pkg/pillar/cas/containerd.go

```diff
@@ -705,6 +708,85 @@ func (c *containerdCAS) PrepareContainerRootDir(rootPath, reference, rootBlobSha
 		logrus.Errorf(err.Error())
 		return err
 	}
+	// On kubevirt eve we also write the mountpoints and cmdline files to the rootPath.
+	// Once rootPath is populated with all necessary files, this rootPath directory will be
+	// converted to a PVC and passed in to domainmgr to attach to VM as rootdisk
+	// NOTE: For kvm eve these files are generated and passed to bootloader in pkg/pillar/containerd/oci.go:AddLoader()
```

Comment:
Is the pillar/cas package the best place to do this for kubevirt? Or would it make more sense to do it in a structurally similar place e.g., in the pillarcontainerd package?

Reply:
I believe pillar/cas is the best place for kubevirt eve. Because we need to have a rootfs directly laid out with all files before we convert to PVC. After that only PVC will be used in case of kubevirt eve. For kvm eve doing in pkg/pillar/containerd/oci.go:AddLoader() makes sense because it uses OCI spec.